r/PACKAGING.md

```diff
@@ -18,86 +18,71 @@
-- [ ] Regenerate arrow_X.X.X.tar.gz (i.e., `make build`)
-
-## Check binary Arrow C++ distributions specific to the R package
-
-- [ ] Upload the .tar.gz to [win-builder](https://win-builder.r-project.org/upload.aspx) (r-devel only)
-  and confirm (with Jon, who will automatically receive an email about the results) that the check is clean.
-- [ ] Upload the .tar.gz to [MacBuilder](https://mac.r-project.org/macbuilder/submit.html)
-  and confirm that the check is clean
-- [ ] Check `install.packages("arrow_X.X.X.tar.gz")` on Ubuntu and ensure that the
-  hosted binaries are used
-- [ ] `devtools::check_built("arrow_X.X.X.tar.gz")` locally one more time (for luck)
-
-## CRAN submission
-- [ ] Upload arrow_X.X.X.tar.gz to the
-  [CRAN submit page](https://xmpalantir.wu.ac.at/cransubmit/)
-- [ ] Confirm the submission email
-
-Wait for CRAN...
-- [ ] Accepted!
-- [ ] Tag the tip of the CRAN-specific release branch with `r-universe-release`
-- [ ] Add a new line to the matrix in the [backwards compatability job](https://github.com/apache/arrow/blob/main/dev/tasks/r/github.linux.arrow.version.back.compat.yml)
-- [ ] (patch releases only) Update the package version in `ci/scripts/PKGBUILD`, `dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb`, `r/DESCRIPTION`, and `r/NEWS.md`
-- [ ] (CRAN-only releases) Rebuild news page with `pkgdown::build_news()` and submit a PR to the asf-site branch of the docs site with the contents of `arrow/r/docs/news/index.html` replacing the current contents of `arrow-site/docs/r/news/index.html`
-- [ ] (CRAN-only releases) Bump the version number in `r/pkgdown/assets/versions.json`, and update this on the [the `asf-site` branch of the docs site](https://github.com/apache/arrow-site) too.
-- [ ] Update the packaging checklist template to reflect any new realities of the
-  packaging process.
-- [ ] Wait for CRAN-hosted binaries on the
-  [CRAN package page](https://cran.r-project.org/package=arrow) to reflect the
-  new version
-- [ ] Tweet!
+- [ ] Regenerate arrow_X.X.X.tar.gz (i.e., `make build`).
+
+## Check Binary Arrow C++ Distributions Specific to the R Package
+
+- [ ] Upload the .tar.gz to [win-builder](https://win-builder.r-project.org/upload.aspx) (r-devel only) and confirm with Jon (who will automatically receive an email about the results) that the check is clean.
+- [ ] Upload the .tar.gz to [macOS Builder](https://mac.r-project.org/macbuilder/submit.html) and confirm that the check is clean.
+- [ ] Check `install.packages("arrow_X.X.X.tar.gz")` on Ubuntu and ensure that the hosted binaries are used.
+- [ ] `devtools::check_built("arrow_X.X.X.tar.gz")` locally one more time (for luck).
+
+## Submit Package to CRAN
+
+_This step must be done by the current package maintainer._
+
+- [ ] Upload arrow_X.X.X.tar.gz to the [CRAN submit page](https://xmpalantir.wu.ac.at/cransubmit/).
```

Comment:
so weird that the form is just this page as an iframe on the cran page...

Reply:
Yeah, it doesn't exactly instill confidence.